setup.py

```diff
@@ -42,10 +42,10 @@
         # and the index.rst file in the docs when you change these
         "numpy>=1.20",
         "scipy>=1.6.0",
-        "openmdao>=3.10",
+        "openmdao >=3.21, <=3.30",
```

Comment:
Why did you add the upper bound here?

Reply:
I describe the problem in #65. The OM update is the reason GHA failed on my first commit to address your comments. I don’t have an elegant and backward compatible solution. If you do, I’d like to hear it, but that should be a separate PR.